Documentation/metrics.md

```diff
@@ -14,6 +14,18 @@ The metrics under the `etcd` prefix are for monitoring and alerting. They are st
 
 Metrics that are etcd2 related are documented [here][v2-http-metrics].
```

Comment:
@heyitsanthony For your mdlint: https://github.com/coreos/docs/blob/master/STYLE.md#naming

`Metrics specific to etcd2 are documented in the [v2 metrics guide][v2-http-metrics].`